app/templates/components/dialogs/prioritize-build-modal.hbs

```diff
@@ -4,13 +4,17 @@
   @onClose={{this.onClose}}
 >
   <div class="repo-actions-modal__top">
-    <h3 class="repo-actions-modal__header">
+    <h3
+      class="repo-actions-modal__header"
+      prioritize-modal-header
```

Comment:
This wouldn't work unfortunately. Custom attributes will make HTML invalid, it's `data-*` attributes that are aimed for custom HTML attributes (see [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*)). Also `ember-test-selectors` addon removes all attributes starting with `data-test-*` specifically, so it won't clean up these custom attributes. Please see the [addon's documentation](https://github.com/simplabs/ember-test-selectors#ember-test-selectors) to better understand how to use it.